go.work.sum

```diff
@@ -1,37 +1,61 @@
+cel.dev/expr v0.16.0 h1:yloc84fytn4zmJX2GU3TkXGsaieaV7dQ057Qs4sIG2Y=
```

Comment:
Are these changes meant to be included?

Reply:
It's getting updated automatically. I'll create a separate PR for lockfile maintenance.